dtfs-central-api/src/cron-scheduler-jobs/delete-acbs-durable-function-logs/index.ts

```diff
@@ -0,0 +1,18 @@
+import { asString, CronSchedulerJob } from '@ukef/dtfs2-common';
+import { WriteConcernError } from '../../errors';
+import { deleteAllAcbsDurableFunctionLogs } from '../../services/repositories/durable-functions-repo';
+
+const { ACBS_DURABLE_FUNCTIONS_LOG_DELETION_SCHEDULE } = process.env;
+
+const deleteAcbsDurableFunctionLogs = async (): Promise<void> => {
+  const { acknowledged } = await deleteAllAcbsDurableFunctionLogs();
+  if (!acknowledged) {
+    throw new WriteConcernError();
+  }
+};
+
+export const deleteAcbsDurableFunctionLogsJob: CronSchedulerJob = {
+  cronExpression: asString(ACBS_DURABLE_FUNCTIONS_LOG_DELETION_SCHEDULE, 'ACBS_DURABLE_FUNCTIONS_LOG_DELETION_SCHEDULE'),
+  description: 'Deletes durable function logs that have the type "ACBS"',
```

Comment:
I think this has been missed out in the requirements but we should only delete those entries where status is `Completed`. Otherwise we are in the risk of deleting entries which are currently being processed.

* Can the description please be updated.

![image](https://github.com/UK-Export-Finance/dtfs2/assets/32125108/f7aa7074-57a4-48dd-bdfa-b757f6d56b51)